libdevcore/CommonData.h

```diff
@@ -302,17 +281,6 @@ template <class T, class U> std::vector<T> operator+(std::vector<T> _a, U const&
 	return _a += _b;
 }
 
-/// Concatenate two vectors of elements.
-template <class T>
-inline std::vector<T> operator+(std::vector<T> const& _a, std::vector<T> const& _b)
-{
-	std::vector<T> ret(_a);
-	return ret += _b;
-}
-
-
-/// Make normal string from fixed-length string.
-std::string toString(string32 const& _s);
```

Comment:
This `string32` from `libdevcore/Common.h` looks to be not used anywhere

Reply:
That's the reason for removal

Reply:
But you didn't remove `string32` itself?

Reply:
Mistake